components/cards/default/card.hbs

```diff
@@ -0,0 +1,19 @@
+<div class="section--bg-white demo-section">
+  <div class="content-block">
+    <img
+      class="content-block__image"
+      src="https://via.placeholder.com/535x301"
+      alt=""
+    />
+    <date class="content-block__date">
```

Comment:
This should be a `<time>` element, and include a `datetime="2021-06-10"`property